sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -4,11 +4,7 @@
 
 ### Features Added
 
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added support for service version 2023-11-03.
```

Comment:
I don't think this is a change compare with last JS SDK release, as last JS SDK should also be on 2023-11-03.
The main change should be upgrade to CoreV2.
And should we mention API view changes if any?